nemo/collections/asr/modules/transformer/transformer_decoders.py

```diff
@@ -63,6 +68,9 @@ def __init__(
         self.layer_norm_3 = nn.LayerNorm(hidden_size, eps=1e-5)
         self.third_sub_layer = PositionWiseFF(hidden_size, inner_size, ffn_dropout, hidden_act)
 
+        # Information for the adapter module mixin
+        self.self_attention_model = "transf_abs"
```

Comment:
any reason for hard coding `self.self_attention_model`?

Reply:
This is slightly a hack, the adapter forward looks up the type of the attention model to see what kind of adapter to call (and what arguments it takes). It just ignores it if its not `rel_pos_attention` or 'local_rel_pos_attention` or some such variants